### ISSUE TITLE
Terms and conditions

### ISSUE THREAD

**Issue by gavinwye** [hmrc/design-patterns#121]
Terms and conditions of services are not consistent. Depending on the link you click you get a different version of terms and conditions some are from GOV.UK some are from HMRC.

We should standardise terms and conditions across our services so that they are easy for users to understand them if they need to be agreed to.

Content designers often strip out a lot of information contained in terms and conditions, replacing them with easier to understand information.

## Start point

Terms and conditions should be:

- short & concise
- written in plain English
- follow the GOV.UK content style guide

## Teams currently working on this at HMRC

- SDES
- subscriptions
- API team

## Research questions

- do we expect users to read terms and conditions?
- if we don't expect terms and conditions to be read by users who are they there for?

**Comment by olirose** [comment]
Here are the 'terms of participation' from the MTDfB sign up journey for both individuals and agents. These were signed off by the Deputy Director for MTDfB (policy and legal). 

Individual
![image](https://user-images.githubusercontent.com/31846170/31123261-0e6e007e-a837-11e7-8733-f10bab9c5935.png)

Agent
![image](https://user-images.githubusercontent.com/31846170/31123344-6fe49b60-a837-11e7-9f77-1d5117d97219.png)

**Comment by stevenaproctor** [comment]
@gavinwye Similar to #73.

**Comment by gavinwye** [comment]
@stevenaproctor yes similar but no the same

**Comment by jennifer-hodgson** [comment]
This is on the [GOV.UK Design System backlog](https://github.com/alphagov/govuk-design-system-backlog/issues/102).

**Comment by mikeash82** [comment]
Moved to [Terms and conditions](https://github.com/alphagov/govuk-design-system-backlog/issues/102) issue on GOV.UK issues backlog
